modules/cudf/test/cudf-series-test.ts

```diff
@@ -350,3 +350,24 @@ describe.each([new Int32, new Float32, new Float64])('Series.sequence({type=%p,,
     expect([...col.toArrow()]).toEqual([0, 2, 4, 6, 8, 10, 12, 14, 16, 18]);
   });
 });
+
+test('Series.value_counts', () => {
+  const s      = Series.new({type: new Int32, data: [0, 1, 1, 2, 2, 2]});
+  const result = s.value_counts();
+  const count  = [...result.count.toArrow()];
+  const value  = [...result.value.toArrow()];
+
+  expect(count[value.findIndex((el) => el === 0)]).toBe(1);
+  expect(count[value.findIndex((el) => el === 1)]).toBe(2);
+  expect(count[value.findIndex((el) => el === 2)]).toBe(3);
```

Comment:
I guess the order is not guaranteed? would it be possible to sort `result` first so that `count` and `value` can just be compared to concrete expected values?

Reply:
Yeah, that seems like a better approach. Do you have any thoughts on how this might be implemented? If `value_counts` returned a DataFrame, it seems like `sortValues` may work. But since `value_counts` returns an object ([src](https://github.com/rapidsai/node-rapids/blob/b2a14bb3bef071cb3a4431c90c45d0317fb85499/modules/cudf/src/series.ts#L503-L506)) with 2 separate series, I'm not sure the best way to go about this. Paul and I tried to make `value_counts` return a DataFrame, but we ran into issues with circular references in TypeScript since a DataFrame consists of Series.

Reply:
TBH I don't even think the sort is required? just iterating over the indices [0, 1, 2] would do. As another suggestion: maybe change the values in the original data to be things that are clearly neither indices, nor counts? e.g. 
```
Series.new({type: new Int32, data: [1000, 1100, 1100, 1200, 1200, 1200]});
```
(Probably would not hurt to "un-sort" those values too, unless that is an assumption of the method)

Reply:
updated in 239f39a